Selenium.psm1

```diff
@@ -27,6 +28,117 @@ if($IsLinux -or $IsMacOS){
     }
 }
 
+
+#region htmlagilitypack
+
+function Init-HTMLAgilityPack {
+    [CmdletBinding()]
+    param ($Element)
+    
+    # Initialize the Selenium Driver
+    $driver = Init-SeDriver -Element $Element
+    Refresh-HTMLAgilityPackSource -Driver $driver
+
+}
+
+function Refresh-HTMLAgilityPackSource {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        $Driver
+    )
+
+    process {
+        # If the Driver Page Source is $null, skip
+        if ($Null -eq $Driver.PageSource) {
+            $Driver.HTMLAgilityPackDocument = $null 
+        } else {
+            $Driver.HTMLAgilityPackDocument.LoadHtml($Driver.PageSource)
+        }
+
+    }
+
+}
+
+function Get-SeHTMLAgilityPack {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        $Driver
+    )
+        
+    process {
+        return $Driver.HTMLAgilityPackDocument
+    }
+    
+}
+
+function Refresh-SeHTMLAgilityPack {
+    [CmdletBinding()]
+    param (
+        [Parameter(Mandatory)]
+        $Driver
+    )
+
+    process {
+        Refresh-HTMLAgilityPackSource -Driver $Driver
+    }
+    
+    end {
+
+        if ($null -eq $Driver.HTMLAgilityPackDocument) {
+            Throw ([System.InvalidOperationException]::new("Missing Selenium HTML Driver. Please Start the Selenium Driver prior to invoking the HTMLAgilityPack."))
+        }
+
+        return $Driver.HTMLAgilityPackDocument
+
+    }
+
+}
+
+function Get-SeChildElements {
```

Comment:
In general, I think using singular nouns for cmdlets is the typical recommendation?
```suggestion
function Get-SeChildElement {
```

Reply:
Sounds good!